Source/UseCases/Implementations/DefaultOnboardingUseCase.swift

```diff
@@ -26,6 +26,10 @@ extension DefaultOnboardingUseCase: OnboardingUseCase {
         return preferences.isTrue(.hasAcceptedTermsOfService)
     }
 
+    var hasOptedForAnalyticsPermissions: Bool {
+        return preferences.hasValue(for: .hasOptedForAnalyticsPermissionsAcceptingTracking)
```

Comment:
let me suggest splitting this into two different key-value:
`hasAnsweredAnalyticsPermissionsQuestion` which only is used by the OnboardingCoordinator determining if the scene should be showed or not

and then another key-value
`userAcceptsAnalyticsTracking` which only sets to true if the user accepts analytics, which is used by the `Tracker` to determine if we are allow to send analytics or not.